src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/GithubAuthenticator.kt

```diff
@@ -56,11 +59,11 @@ class GithubAuthenticator(private val userRepository: UserRepository,
 
         if (!githubOrg.isEmpty() && !userBelongToOrg(githubOrg, user))
         {
-            throw CredentialsException("User is not a member of GitHub org $githubOrg or token does not include read:org scope")
+            throw loggedCredentialsException("User is not a member of GitHub org $githubOrg or token does not include read:org scope")
```

Comment:
Just an opportunistic refactor as I wanted to see these errors logged.

Reply:
Could there be a more general exception logging solution at the app level, which handles any uncaught exceptions, logs them, then rethrows?

Reply:
We have this already, the `ErrorHandler` class, but these are handled rather than unhandled exceptions so don't make it that far. But looks like `pac4j` is already logging them so don't actually know why I though it wasn't! My bad, looks like this code is totally superfluous, will back it out.